BrowserKit/Sources/ToolbarKit/ToolbarManager.swift

```diff
@@ -31,11 +31,11 @@ public class DefaultToolbarManager: ToolbarManager {
         // - the toolbar is displayed at the bottom
         // display the bottom border if
         // - the toolbar is displayed at the top and the website was scrolled
-        // - we are in private mode
+        // - the toolbar is displayed at the top and we are in private mode
         if borderPosition == .top {
             return toolbarPosition == .bottom
```

Comment:
based on the comments, what do you think about the following? It seems to be easier to read and to me its self documented and probably can remove the comments. Overall, instead of working through the booleans, we can get the position directly.

```
public func getAddressBorderPosition(for toolbarPosition: AddressToolbarPosition,
                                           isPrivate: Bool,
                                           scrollY: CGFloat) -> AddressToolbarBorderPosition? {
           if toolbarPosition == .bottom {
              return .top
           else if toolbarPosition == .top && (scrollY > 0 || isPrivate)
              return toolbarPosition == .bottom
           else
              return nil
}                                           
```

Reply:
Great idea! Thanks so much for suggesting this 🙏🏼 I refactored the code accordingly. Definitely looks cleaner and more readable this way.

Reply:
Really like how it turned out, thanks for refactoring 😄